pkg/eks/client.go

```diff
@@ -40,6 +40,7 @@ func (c *ClusterProvider) NewClient(spec *api.ClusterConfig) (*Client, error) {
 	client := &Client{
 		Config: config,
 	}
+	// TODO: usage of STS v2 is blocked on sigs.k8s.io/aws-iam-authenticator/pkg/token
```

Comment:
>And its blocked because sigs.k8s.io/aws-iam-authenticator/pkg/token uses the V1 SDK?

Correct. 

>It would be good to expand on why its blocked and is there a follow-up issue to re-visit this again in the future?

Yeah, we are going to open a follow-up issue: https://github.com/weaveworks/eksctl/pull/4926#pullrequestreview-915916354

Reply:
Be good to update the comment and include the issue number in the TODO?

Reply:
Agreed, done.